multinode_runfabtests.sh

```diff
@@ -3,25 +3,16 @@
 check_kernel_has_fork_support()
 {
     host=$1
-    kernel_version=$(ssh $host uname -r)
-    older=$(printf "$kernel_version\n5.13.0" | sort -V | head -n 1)
-    if [ $older = "5.13.0" ]; then
+
+    scp fork_checker.c $host:~
```

Comment:
This `multinode_runfabtests.sh` is executed on remote host, you would have to scp this fork_checker.c to the remote host before this whole script is executed. See this https://github.com/awslabs/libfabric-ci-scripts/blob/master/multi-node.sh#L24-L25